.github/workflows/coverage.yml

```diff
@@ -0,0 +1,36 @@
+# Based on https://github.com/actions-rs/meta/blob/master/recipes/quickstart.md
+
+# Do this on every push, but don't double up on tags
+on:
+  push:
+    tags-ignore:
+      - "**"
+    branches:
+      - "**"
+  pull_request:
+
+env:
+  MWA_ASVO_API_KEY: ${{ secrets.MWA_ASVO_API_KEY }}
+
+name: Generate Coverage report
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    container: mwatelescope/giant-squid:latest
```

Comment:
I personally think it's easier to not depend on a container, but your choice

Reply:
@d3v-null don't suppose you can comment on this? Is a container necessary here?

Reply:
According to Dev, this is to run the coverage report in an image with all of the dependencies installed, reducing the total CI time.